roles/rciam-probes/tasks/install-RedHat.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Ensure Repos are installed (RedHat)
+  yum_repository:
+    name: "{{ item.name }}"
+    description: "{{ item.description }}"
+    file: "{{ item.file }}"
+    baseurl: "{{ item.baseurl }}"
+    gpgcheck: "{{ item.gpgcheck}}"
+    enabled: "{{ item.enabled }}"
+  loop: "{{ repos | default([])}}"
+  become: yes
+  notify: yum Update
```

Comment:
Is this necessary? See comment about yum update handler below

Reply:
removed